datasources/helpscout/hs_get_tags.py

```diff
@@ -0,0 +1,40 @@
+from helpscout.client import HelpScout
+import pandas as pd
+from sqlalchemy import create_engine
+from classes.alchemyClasses import HS_tag
+import sys
+import pendulum
+from sqlalchemy.orm import Session
+from sqlalchemy.ext.declarative import declarative_base
+Base = declarative_base()
+
+
+#from dbconfig import helpscout_api, local_db
+
+def create_HS_engine(engine, hs_app_id, hs_app_secret):
+    hs = HelpScout(app_id=hs_app_id, app_secret=hs_app_secret, sleep_on_rate_limit_exceeded=True)
+    return hs, engine
+
+def update_tags(engine, hs_app_id, hs_app_secret, dis, die):
+
+    hs, engine = create_HS_engine(engine, hs_app_id, hs_app_secret)
+
+    #Importem totes les tags de Som Energia a HelpScout
+    tags = hs.tags.get()
+
+    #per tema idempotencia nomes les mes noves de l'ultima execucio
+    tags_insert = [HS_tag(id=t.id, name=t.name) for t in tags if pendulum.parse(t.createdAt) <= die and pendulum.parse(t.createdAt) > dis]
```

Comment:
millor posar `dis < pendulum.parse(t.createdAt) and pendulum.parse(t.createdAt) <= die]`
queda més clar que el rang és de dis a die

Reply:
Hola Roger, he mirat la documentació i he provat i no veig com filtrar els tags, si pots m'ho expliques algun dia.